bot.py

```diff
@@ -30,8 +31,9 @@ async def on_ready():
     token = json.loads(data_file.read())['TOKEN']
 
 bot.remove_command('help')  # Overwrite normal 'help' command
-#bot.add_cog(CommandErrorHandler(bot))
+bot.add_cog(CommandErrorHandler(bot))
 bot.add_cog(MemberStats(bot))
+bot.add_cog(MemberRoles(bot))
```

Comment:
As this function is not quite ready to be used on the Discord server yet, perhaps it should not be added as a cog quite yet. Alternatively, we can give it some kind of on/off switch.